### PR TITLE
Add column arn for the table alicloud_ecs_instance. Closes #142

### LINKED ISSUE #142: Add column arn for the table alicloud_ecs_instance
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Required for the query in new format.

**Describe alternatives you've considered**
N/A

**Additional context**
https://www.alibabacloud.com/help/doc-detail/172061.htm#title-q22-kas-ids

### PR DESCRIPTION
**Pull request by rajeshbal65**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
### Basic Instance Info
```sql
select
  instance_id,
  name,
  arn,
  status,
  instance_type,
  os_name_en,
  public_ip_address,
  private_ip_address,
  zone
from
  alicloud_ecs_instance;

+------------------------+-------------------------+-------------------------------------------------------------------------+---------+---------------+------------------------------------+-------------------+--------------------+-------------+
| instance_id            | name                    | arn                                                                     | status  | instance_type | os_name_en                         | public_ip_address | private_ip_address | zone        |
+------------------------+-------------------------+-------------------------------------------------------------------------+---------+---------------+------------------------------------+-------------------+--------------------+-------------+
| i-a2dh6ihk2z51yjre9mdn | launch-advisor-20210609 | arn:acs:ecs:ap-south-1:5982111499156037:instance/i-a2dh6ihk2z51yjre9mdn | Running | ecs.g6.large  | Alibaba Cloud Linux  3.2104 64 bit | ["147.139.5.186"] | ["172.18.242.246"] | ap-south-1b |
+------------------------+-------------------------+-------------------------------------------------------------------------+---------+---------------+------------------------------------+-------------------+--------------------+-------------+
```
```
</details>

### CODE REVIEW

File: alicloud/table_alicloud_ecs_instance.go [hunk 1]
```diff
@@ -429,8 +436,8 @@ func tableAlicloudEcsInstance(ctx context.Context) *plugin.Table {
 				Name:        "akas",
 				Description: ColumnDescriptionAkas,
 				Type:        proto.ColumnType_JSON,
-				Hydrate:     getEcsInstanceAka,
-				Transform:   transform.FromValue(),
+				Hydrate:     getEcsInstanceArn,
+				Transform:   transform.FromValue().Transform(ensureStringArray),
```
Comment (Subhajit97) [inline alicloud/table_alicloud_ecs_instance.go:440]:
```suggestion
				Transform:   transform.FromValue().Transform(transform.EnsureStringArray),
```

File: alicloud/table_alicloud_ecs_instance.go [hunk 2]
```diff
@@ -543,8 +550,8 @@ func getEcsInstance(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateD
 	return nil, nil
 }
 
-func getEcsInstanceAka(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
-	plugin.Logger(ctx).Trace("getEcsInstanceAka")
+func getEcsInstanceArn(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```
Comment (Subhajit97) [inline alicloud/table_alicloud_ecs_instance.go:553]:
```suggestion
func getEcsInstanceARN(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

**Review by Subhajit97 — CHANGES_REQUESTED** [review]
@rajeshbal65 Please see comments, thanks!
Can you also add the integration test to validate the `arn` column?

**Review by Subhajit97 — APPROVED** [review]
LGTM

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**